Workbooks/CosmosDb/Resource Insights/Resource Insights.workbook

```diff
@@ -69,17 +69,65 @@
             "value": {
               "durationMs": 86400000
             }
+          },
+          {
+            "id": "82f8fb72-8e01-4516-8115-b62082df1e44",
+            "version": "KqlParameterItem/1.0",
+            "name": "Database",
+            "type": 2,
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{Resources}/sqlDatabases\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2020-04-01\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.value.*.properties.resource.id\",\"columns\":[]}}]}",
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "defaultValue": "value::1",
+            "queryType": 12,
+            "value": "Demo"
           }
         ],
         "style": "above",
-        "queryType": 0,
-        "resourceType": "microsoft.resourcegraph/resources"
+        "queryType": 12
       },
-      "name": "parameters - 0",
+      "name": "Container",
       "styleSettings": {
         "margin": "0px"
       }
     },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "0abe0edd-1fb0-46fb-b7f2-6c0c21245fda",
+            "version": "KqlParameterItem/1.0",
+            "name": "Container",
+            "type": 2,
+            "description": "In order to view the data at a container level, you must first choose a database. ",
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{Resources}/sqlDatabases/{Database:label}/containers\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2020-04-01\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.value.*.properties.resource.id\",\"columns\":[]}}]}",
+            "value": null,
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "queryType": 12
+          }
+        ],
+        "style": "pills",
+        "queryType": 12
+      },
+      "conditionalVisibility": {
+        "parameterName": "Database",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "parameters - 11"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "{Database}"
```

Comment:
Can take this text out, or can incorporate it into a workbook by formatting this that makes it more obvious that that they are looking at the scope of a particular DB/Container